app/models/miq_request_workflow.rb

```diff
@@ -67,17 +71,15 @@ def self.make_request(request, values, requester_id, auto_approve = false)
     end
   end
 
-  def create_request(values, requester_id, target_class, event_name, event_message, auto_approve = false)
+  def create_request(values, _requester_id, target_class, event_name, event_message, auto_approve = false)
```

Comment:
Copying from commit comment...

If we're never going to use requester_id, should we delete it and update the callers?